### PR TITLE
Add export/import methods

### PR DESCRIPTION
**Pull request by cammellos**
Add 2 convenience methods for debugging:

1) ExportUnencryptedDatabase: -> unencrypts the database and exports it to a directory
2) ImportUnencryptedDatabase: -> imports a cleartext  database an encrypts it

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: api/geth_backend.go [hunk 1]
```diff
@@ -400,6 +400,60 @@ func (b *GethStatusBackend) StartNodeWithAccount(acc multiaccounts.Account, pass
...
+	if b.appDB != nil {
+		return nil
+	}
+	if len(b.rootDataDir) == 0 {
+		return errors.New("root datadir wasn't provided")
+	}
+
+	// Migrate file path to fix issue https://github.com/status-im/status-go/issues/2027
+	oldPath := filepath.Join(b.rootDataDir, fmt.Sprintf("app-%x.sql", acc.KeyUID))
+	newPath := filepath.Join(b.rootDataDir, fmt.Sprintf("%s.db", acc.KeyUID))
```
Comment (Samyoul) [inline api/geth_backend.go:415]: Do we need to pass in a whole `multiaccounts.Account` if we only use the `acc.KeyUID`?

File: api/geth_backend.go [hunk 2]
```diff
@@ -400,6 +400,60 @@ func (b *GethStatusBackend) StartNodeWithAccount(acc multiaccounts.Account, pass
...
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	if b.appDB != nil {
+		return nil
+	}
+	if len(b.rootDataDir) == 0 {
+		return errors.New("root datadir wasn't provided")
+	}
+
+	path := filepath.Join(b.rootDataDir, fmt.Sprintf("%s.db", acc.KeyUID))
```
Comment (Samyoul) [inline api/geth_backend.go:447]: Same as above, it looks like we only use `acc.KeyUID`

**Review by Samyoul — APPROVED** [review]
Zero problems, cool new methods. Just a question about using a struct as a parameter when we only need one of its members.

**Comment by cammellos** [comment]
Closing as merged in a separate PR that has been manually tested by QA

**Pull request closed without merging**